bindings/wrap/polywrap.graphql

```diff
@@ -0,0 +1,49 @@
+type Module {
+    ## model
+    getContextSize(model: String!): UInt!
```

Comment:
NIT: to help with GraphQL's readability, I much prefer having each argument on a new line:
```graphql
getContextSize(
  model: String!
): UInt!
```

We do this in most all other wraps. Can we do this here as well? I think it's easier to parse.